mission/functions/systems/sites/fn_sites_generate.sqf

```diff
@@ -54,13 +54,18 @@ private _fnc_findPos = {
 	//Create initial AA emplacements
 	for "_i" from 1 to (1 + ceil random (vn_mf_s_max_aa_per_zone - 1)) do
 	{
-		[[_center, _sizeMax / 4, _sizeMax / 2] call _fnc_findPos] call vn_mf_fnc_sites_create_aa_site;
+		private _pos = [_center, _sizeMax / 4, _sizeMax / 2] call _fnc_findPos;
+
+		[_pos] call vn_mf_fnc_sites_create_aa_site;
+		[_pos, 5] call vn_mf_fnc_sites_hide_terrain_objects_bbox_intersections;
```

Comment:
Is this part of MF code scheduled? If it is you should hide terrain objects before spawning.